python/llm/src/bigdl/llm/transformers/low_bit_linear.py

```diff
@@ -59,7 +59,7 @@
 from bigdl.llm.ggml.quantize import ggml_tensor_qtype
 IS_SERVER = is_server()
 IS_SPR = is_spr()
-TORCH_LINEAR_THRESHOLD = 96
+TORCH_LINEAR_THRESHOLD = int(os.getenv("TORCH_LINEAR_THRESHOLD", "96"))
```

Comment:
Can we use a larger value @MeouSker77 ?

Reply:
Changed default to 256.

Reply:
You probably should run a grid search on the target machine to determine the default value.

Reply:
> You probably should run a grid search on the target machine to determine the default value.

After a grid search on Icelake platform, we can set the default value to 512. :)